dev/deps/spark-deps-hadoop-2.6

```diff
@@ -144,7 +144,9 @@ metrics-json-3.1.5.jar
 metrics-jvm-3.1.5.jar
 minlog-1.3.0.jar
 netty-3.9.9.Final.jar
-netty-all-4.0.47.Final.jar
+netty-all-4.1.17.Final.jar
+netty-buffer-4.1.17.Final.jar
+netty-common-4.1.17.Final.jar
```

Comment:
@zsxwing do you think `netty-buffer` and `netty-common` can be safely excluded in the Spark pom because the same classes also in `netty-all`?

Reply:
@BryanCutler Yes. It should be safe.

Reply:
Cool, thx just wanted to be sure